localstack/services/awslambda/lambda_executors.py

```diff
@@ -1593,7 +1593,9 @@ def docker_image_for_lambda(cls, lambda_function: LambdaFunction):
         if runtime == "nodejs14.x" and docker_image == DEFAULT_LAMBDA_CONTAINER_REGISTRY:
             # TODO temporary fix until lambci image for nodejs14.x becomes available
             docker_image = "localstack/lambda-js"
-        if runtime == "python3.9" and docker_image == DEFAULT_LAMBDA_CONTAINER_REGISTRY:
+        elif (
+            runtime == "python3.9" or runtime == "dotnet6"
```

Comment:
```suggestion
            runtime in ["python3.9", "dotnet6"]
```
could possibly also work. No real official style guidelines for this though, you might as well leave it.

Reply:
this actually reads better. I updated it